churaverse-plugins-server/src/churarenPlugin/churarenAlchemyItems/alchemyItemPlugins/alchemyItemPlugins.ts

```diff
@@ -1,3 +1,4 @@
 import { BaseAlchemyItemPlugin } from '@churaverse/churaren-alchemy-plugin-server/domain/baseAlchemyItemPlugin'
+import { ExplosionPlugin } from '@churaverse/churaren-explosion-plugin-server'
 
-export const alchemyItemPlugins: Array<typeof BaseAlchemyItemPlugin> = []
+export const alchemyItemPlugins: Array<typeof BaseAlchemyItemPlugin> = [ExplosionPlugin]
```

Comment:
ちょっとしたTipsですが、以下のようにして一行一要素にするとこの先の実装でコンフリクトをなるべく抑えることができます
```suggestion
export const alchemyItemPlugins: Array<typeof BaseAlchemyItemPlugin> = [
  ExplosionPlugin,
]
```

また、このようにコンフリクトしやすい＆追加されることが予めわかっている場合は以下のように最初からコメントアウトしておくorコメントで区切っておくとコンフリクトを回避できます
```suggestion
export const alchemyItemPlugins: Array<typeof BaseAlchemyItemPlugin> = [
  // ExplosionPlugin, // 各PRでそれぞれの行がコメントアウトされるだけなのでコンフリクトしない
  // HealingPotionPlugin,
  // RevivalItemPlugin,
]
```

既に後続PRを作っているので、今回はもう使えないテクニックですが...